db/structure.sql

```diff
@@ -30,6 +30,39 @@ SET default_tablespace = '';
 SET default_with_oids = false;
 
 --
+-- Name: journal_entries; Type: TABLE; Schema: public; Owner: -; Tablespace: 
+--
+
+CREATE TABLE journal_entries (
+    id integer NOT NULL,
+    procurement_area_id integer NOT NULL,
+    total_slots integer NOT NULL,
+    start_time timestamp without time zone NOT NULL,
+    end_time timestamp without time zone,
+    status character varying
+);
+
+
+--
+-- Name: journal_entries_id_seq; Type: SEQUENCE; Schema: public; Owner: -
+--
+
+CREATE SEQUENCE journal_entries_id_seq
```

Comment:
Ah this is the sequence in question. Not sure how to get rid of it? Never used this `structure.sql` milarky.